### PR TITLE
feat(wokwi): Add scenario path parameter

### PR DESCRIPTION
**Pull request by P-R-O-C-H-Y**
This PR adds a support of scenario file for Pytest-wokwi.
To use scenario file add `--wokwi_scenario <path>` to the pytest command

### CODE REVIEW

**Comment by P-R-O-C-H-Y** [comment]
@hfudev Can you please take a look?

File: pytest-embedded-wokwi/pytest_embedded_wokwi/wokwi_cli.py [hunk 1]
```diff
@@ -57,6 +58,8 @@ def __init__(
         cmd = [wokwi_cli, '--interactive', app.app_path]
         if (wokwi_timeout is not None) and (wokwi_timeout > 0):
             cmd.extend(['--timeout', str(wokwi_timeout)])
+        if (wokwi_scenario is not None) and os.path.exists(wokwi_scenario):
+            cmd.extend(['--scenario', wokwi_scenario])
```
Comment (hfudev) [inline pytest-embedded-wokwi/pytest_embedded_wokwi/wokwi_cli.py:62]: Not sure about it, when enabling `--scenario ...`, we should not use `--interactive`, right?

**Review by hfudev — COMMENTED** [review]
LGTM. only one question

File: pytest-embedded-wokwi/pytest_embedded_wokwi/wokwi_cli.py (hunk 1 above)
Comment (P-R-O-C-H-Y) [reply pytest-embedded-wokwi/pytest_embedded_wokwi/wokwi_cli.py:62]: `--interactive - Redirect stdin to the simulated serial port`
I run multiple tests as it is with the `--interactive` set, and all worked well. Pytest with expecting some output or with UNITY inside the sketch.

Comment (hfudev) [reply pytest-embedded-wokwi/pytest_embedded_wokwi/wokwi_cli.py:62]: cool. no problem then.

**Comment by hfudev** [comment]
(no need to rebase then, I'll merge this PR when 3.7 test passed since we're currently lack of wokwi test job)

**Comment by P-R-O-C-H-Y** [comment]
> (no need to rebase then, I'll merge this PR when 3.7 test passed since we're currently lack of wokwi test job)

ok Thanks :)

**Pull request merged**